config.py

```diff
@@ -21,3 +21,9 @@
 # limits on the size of the JSON doc stored for user preferences
 MAX_ALLOWED_JSON_SIZE = 1000
 MAX_ALLOWED_JSON_KEYS = 100
+
+# user_id for anonymous users - fix this
+BOOTSTRAP_USER_ID = None
```

Comment:
Nah, more like "fix in local_config or the yaml before deployment" since the bootstrap user ID shouldn't be exposed - I'll make that comment more clear.